client/live.js

```diff
@@ -1,8 +1,32 @@
 var $ = require("jquery");
-var io = require("socket.io-client");
+var SockJS = require("sockjs-client");
 var stripAnsi = require('strip-ansi');
 require("./style.css");
 
+var sock = null;
+var hot = false;
+var currentHash = "";
+
+var newConnection = function(handlers) {
+	sock = new SockJS('/sockjs-node');
```

Comment:
Just wanted to flag that this breaks certain usage that relied on the previous `/socket.io` path. They now have to be `/sockjs-node`. Should this have been a major version bump, according to semver?